### ISSUE TITLE
does not create deb file but directory instead

### ISSUE THREAD

**Issue by flip111** [AntumDeluge/debreate#12]
Hi, i wanted to create a package for mercury and name the deb file like `mercury_14.01.1_amd64.deb` instead i get a directory like `mercury_14.01.1_amd64__dbp__` with what looks like could be the contents of the deb file.

**Comment by AntumDeluge** [comment]
What version of Debreate are you using? That is the stage directory where the files are stored temporarily for creating the .deb package. But, it looks like an error is occurring before that causing the package build to fail & leave the staged directory.

**Comment by flip111** [comment]
I use debreate 0.7.13. On my second attempt i ran out of disc space, so that made me look at the first folder it had created, which is unexpectedly very big. The files i want in the package are `5253 items, totalling 1,0 GB`, while the created temporary directory is `445686 items, totalling 8,5 GB`. I investigated where the size difference comes from and found the following:

in the source directory `~/programs/mercury/lib/mercury/ints/reg.gc/Mercury` there is a symbolic link `opts -> ..`
Why? i don't know it's just the way it was installed by `make install`

debreate seems unaware that it is a symbolic link because it just created a normal directory instead and keeps copying all the files until some recursion limit (in the directory tree) or running out of disc space.

Can debreate not handle symbolic links??

**Comment by AntumDeluge** [comment]
No, it currently cannot handle symbolic links. But, I don't think it would be too hard fix. I will add it to my TODO list.

**Comment by AntumDeluge** [comment]
Added to TODO list: 948667c

**Comment by AntumDeluge** [comment]
Should be fixed with commit 68533a3. Although, it may not work for nested symbolic links. Will be releasing a new testing version soon.
